playbooks/common/private/components.yml

```
@@ -64,3 +64,6 @@
 
 - import_playbook: ../../openshift-cluster-autoscaler/private/config.yml
   when: openshift_cluster_autoscaler_install | default(false) | bool
+
+- import_playbook: ../../openshift-noobaa/private/config.yml
+  when: openshift_noobaa_install | default(true) | bool
```

```suggestion
  when: openshift_noobaa_install | default(false) | bool
```
I don't think it should be installed by default though